app/user.go

```diff
@@ -29,91 +29,140 @@ func (a *App) UserDashboardHandler(c echo.Context) error {
 		return err
 	}
 
-	log.Debug("getting instant launch items")
+	// Fetch instant launches
 	ilAPI, err := apis.NewInstantLaunchesAPI(a.config)
 	if err != nil {
 		log.Error(err)
 		return err
 	}
-	ilItems, err := ilAPI.PullItems(ctx)
+
+	ilChan := make(chan []map[string]interface{})
+	ilErrChan := make(chan error)
+
+	go ilAPI.PullItemsAsync(ctx, ilChan, ilErrChan)
```

Comment:
Here and below is where the actual changes happen. By and large, we're changing things like:

```go
foo, err := something.SomeMethod(ctx, ... some other stuff maybe)
if err != nil {
  log.Error(err)
  return err
}
```

and replaces it with something like

```go
oneChan := make(chan ...whatever return value)
errChan := make(chan error)

go something.SomeMethodAsync(ctx, oneChan, errChan, ...other stuff)

... much further down ...

err = <- errChan
if err != nil {
  log.Error(err)
  return err
}
foo := <-oneChan
```

This lets the goroutine process in the background. Most of the derefs are right before the return at the end, with two exceptions: public app IDs are needed to set off a variety of calls, and featured app IDs are needed for the popular featured app call, so those two are dereferenced as late as they can be before they're needed.